Add Browserless Log In Section to Log in to Astro (#1028)
astronomer/docs: astro/log-in-to-astro.md

    Confirmation messages appear in the Cloud UI and in the Astro CLI indicating that your login was successful and that your computer is now connected. The name of your default Workspace in the Astro CLI also appears. To switch Workspace contexts after you log in, see [astro workspace switch](https://docs.astronomer.io/astro/cli/astro-workspace-switch).

## Browserless authentication

The following options are available if you're unable to use a browser for authnentication:

> 

Reviewer: ```suggestion

- Run `astro login -t` to login with an authentication token. You can get an auth token from https://cloud.astronomer.io/token on a separate machine. Use this option if you must login through a browser because you are using identity provider (IdP) for account authentication.
```

> - Run astro `login -l` in the Astro CLI to manually access the Cloud UI, copy the returned URL, and then run `curl -u <username>:<password> <returned URL>`. Replace `<username>` and `<password>` with your username and password, and `<returned URL>` with the value displayed after running `login -l`.

Reviewer 2: ```suggestion
- Run `astro login -l` to retrieve a URL for logging in using the Cloud UI. Copy this URL. In a separate terminal session, run `curl -u <user-email>:<password> <returned-url>. Note that this option does not work for accounts that are authenticated through an identity provider (IdP).
```
@sunkickr Is this right? I was able to use the link in a different browser and then open the callback link (`http://localhost:12345/callback?code...`) in my main browser to finish authentication with an IdP account, but I don't know how a browserless computer could open that callback link

    Confirmation messages appear in the Cloud UI and in the Astro CLI indicating that your login was successful and that your computer is now connected. The name of your default Workspace in the Astro CLI also appears. To switch Workspace contexts after you log in, see [astro workspace switch](https://docs.astronomer.io/astro/cli/astro-workspace-switch).

## Browserless authentication

The following options are available if you're unable to use a browser for authnentication:

- Run `astro login -l` to retrieve a Cloud UI log in URL and then copy the URL. In a separate terminal session, run `curl -u <user-email>:<password> <returned-url>`. This option doesn't work if you're using an identity provider (IdP) for account authentication.

> - Run `astro login -t` if you're using an IdP for account authentication and you want to log in with an authentication token. To obtain an authentication token, on a separate machine browse to <https://cloud.astronomer.io/token>. 

Reviewer 2: ```suggestion
- Run `astro login -t` to log in with an authentication token. To obtain an authentication token on a separate machine, open `https://cloud.astronomer.io/token`.
```
This seems to be the most versatile way to log in browserless, so I would put it first in the list
